.github/workflows/push-deploy-website.yml

```
@@ -0,0 +1,57 @@
+jobs:
+  Deploy-Docusaurus:
+    name: Deploy https://docs-admin.talawa.io website
+    runs-on: ubuntu-latest
+    # Run only if the develop branch and not dependabot
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    environment:
+      # This "name" has to be the repos' branch that contains 
+      # the current active website. There must be an entry for
+      # the same branch in the PalisadoesFoundation's
+      # "Code and automation > Environments > gigithub-pages"
+      # menu. The branch "name" must match the branch in the 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)